server/controllers/adminController/index.js

```diff
@@ -0,0 +1,21 @@
+import { collections } from '../../utils/mongoCollections'
+
+const AdminUsersController = {
+  update: async (req, res) => {
+    try {
+      const { appDb } = req.app.locals
+      const { uid } = req.params
+      const updatedUserPermissions = await appDb
+        .collection(collections.users)
+        .updateOne({ uid }, { $set: req.body })
```

Comment:
this could be a good fit on the `UserModel`.

```js
const UserModel = {
  update: async (db, uid, userAttributes) => {
    return await db
      .collection(collections.users)
      .updateOne({ uid }, { $set: userAttributes })
  },
}

Reply:
[8814b2b](https://github.com/AMP-SCZ/dpdash/pull/294/commits/8814b2b15da0a94568320a77593fb7c635f9ffb9)